st2common/st2common/util/config_loader.py

```diff
@@ -60,12 +60,16 @@ def __init__(self, pack_name, user=None):
     def get_config(self):
         result = {}
 
-        # 1. Retrieve values from pack local config.yaml file
-        config = self._config_parser.get_config()
-
-        if config:
-            config = config.config or {}
-            result.update(config)
+        # The use of config.yaml local to the pack is deprecated. The use_local_pack_config
+        # option is default to False and will only be available for a short time in case
+        # some users are caught off guard.
+        if cfg.CONF.content.use_local_pack_config:
```

Comment:
How does this behave if that option is unset? The `if` statement fails? (which is good, if that's what it does)